tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -247,15 +290,18 @@ export class GraphExecutor implements FunctionExecutor {
                 `Please use model.executeAsync() instead.`);
           }
           tensorsMap[node.name] = tensors;
+          this.cloneTensorList(tensors, node.name, this.clonedTensorsMap);
```

Comment:
Does this need a check for `this.keepIntermediateTensors`?

Reply:
No, it doesn't, because `this.clonedTensorsMap` is null, but IMO that's a bit confusing. See my other comment on this.